project.clj

```diff
@@ -59,7 +59,8 @@
    :repl { :plugins [[cider/cider-nrepl "0.10.0-SNAPSHOT"]]
            :repl-options {:init (set! *print-length* 500)}}             
    :dev {
-      :dependencies [[org.omcljs/om "0.9.0"]
+      :dependencies [;[org.omcljs/om "0.9.0"]
```

Comment:
I've actually added that on purpose to remind people that if they're using Om Next they should not require `0.9.0`, but I'm happy to take it away